dbt/models/qc/schema.yml

```diff
@@ -916,6 +892,64 @@ models:
     meta:
       export_name: Class does not equal LUC
 
+  - name: qc.vw_report_town_close_res_parcels_not_set_to_cost_approach
+    description: '{{ doc("view_vw_report_town_close_res_parcels_not_set_to_cost_approach") }}'
+    config:
+      tags:
+        - qc_report_town_close
```

Comment:
**issue:** Won't these tests never get run, since we're querying for `test_qc_*` tags when running the daily build?

Reply:
Good question! Technically `qc_vw_incorrect_val_method_revcode_eq_1` would still get run because I moved the tag over to the test from the model. Still, I think that was a bit unnecessarily defensive of me to do -- thinking about it again, I realize now that we'll probably want any more tests we add to this model to be run as part of the QC test suite -- so I moved it back from the test to the model in b9ac1baa648f2b9e619c48442a22eb541dac5f79.

Reply:
Ah, I had a tired moment and thought we were running the town close QC reports daily along with the rest of the `test_qc_*` tagged stuff. Now I remember we just call them on demand from export script.

Reply:
No worries, it's confusing! I'm hoping the docs will help clear some of this stuff up.